Management/src/ActuatorWeb/README.md

```diff
@@ -0,0 +1,204 @@
+
+> [!TIP]
+> To quickly confirm that Prometheus is able to scrape the application endpoints, check [the target health page](http://localhost:9090/targets)
+
+#### Prometheus
+
+To launch Prometheus with the configuration for this demo, but without using the docker compose file:
+
+1. Open a shell, `cd` to `\Management\src`
+1. `docker run  --rm -it --pull=always -p 9090:9090 -v $PWD/prometheus:/etc/prometheus prom/prometheus`
+
+#### Grafana
+
+To launch Grafana with the configuration for this demo, but without using the docker compose file:
+
+1. Open a shell, `cd` to `\Management\src`
+1. `docker run  --rm -it --pull=always -p 3000:3000 -v $PWD/grafana/config:/etc/grafana -v $PWD/grafana/dashboards:/var/lib/grafana/dashboards grafana/grafana`
+
+For more information about using Grafana dashboards, see the [Grafana documentation](https://grafana.com/docs/grafana/latest/dashboards/use-dashboards/).
+
+## Using Podman
+
+Because Podman is generally compatible with Docker, the only changes necessary to run this sample with Podman is to change the hostname in places where `host.docker.internal` is used.
+
+When running with Podman, update these files to use `host.containers.internal`:
+
+* [ActuatorWeb appsettings.Development.json](./appsettings.Development.json)
+* [ActuatorApi appsettings.Development.json](../ActuatorApi/appsettings.Development.json)
+* [prometheus.yml](../prometheus/prometheus.yml)
+* [Grafana data source definition](../grafana/config/provisioning/datasources/default.yaml)
+
+## Running on Tanzu Platform for Cloud Foundry
+
+1. Run the `cf push` command to deploy from source (you can monitor logs with `cf logs actuator-web-management-sample`)
+    * When deploying to Windows (or to see `git.properties` in the Info actuator response*), binaries must be built locally before push. Use the following commands instead:
+
+      ```shell
+      dotnet publish -r win-x64 --self-contained
+      cf push -f manifest-windows.yml -p bin/Release/net8.0/win-x64/publish
+      ```
+
+      > [!NOTE]
+      > \* These applications use the GitInfo NuGet package to write a `git.properties` if the .git folder is found.
+      > When the staging process runs on Cloud Foundry, that information is not available.
+      > If you want to see git properties returned when the application is running on Cloud Foundry, publish the application before pushing.
+
+1. Copy the value of `routes` in the output and open in your browser
+1. Refer to [ActuatorApi README](../ActuatorApi/README.md#running-on-tanzu-platform-for-cloud-foundry) for additional instructions.
```

Comment:
```suggestion
1. Run the `cf push` command to deploy from source (you can monitor logs with `cf logs actuator-web-management-sample`)
    * When deploying to Windows (or to see `git.properties` in the Info actuator response*), binaries must be built locally before push. Use the following commands instead:

      ```shell
      dotnet publish -r win-x64 --self-contained
      cf push -f manifest-windows.yml -p bin/Release/net8.0/win-x64/publish
      ```

> [!NOTE]
> \* These applications use the GitInfo NuGet package to write a `git.properties` if the .git folder is found.
> When the staging process runs on Cloud Foundry, that information is not available.
> If you want to see git properties returned when the application is running on Cloud Foundry, publish the application before pushing.

2. Copy the value of `routes` in the output and open in your browser
3. Refer to [ActuatorApi README](../ActuatorApi/README.md#running-on-tanzu-platform-for-cloud-foundry) for additional instructions.
```

Reply:
The `*` inside the note looks odd. How about just this:

> When deploying to Windows (or to see `git.properties` in the Info actuator response)

Reply:
updated